public/app/plugins/datasource/prometheus/querybuilder/binaryScalarOperations.ts

```diff
@@ -71,6 +71,14 @@ export const binaryScalarDefs = [
   },
 ];
 
+export const binaryScalarOperatorToOperatorName = binaryScalarDefs.reduce((acc, def) => {
```

Comment:
@grafana/observability-metrics @aocenas Specifically, could you take a look over the changes I made in [this commit](https://github.com/grafana/grafana/pull/44959/commits/afbaeb3edb1dd69574272a8f31a9d3b57511d5dc) please to resolve an issue caused by circular dependencies.

Note: the circular dependencies are still there, I just mitigated an issue caused by it.